### ISSUE TITLE
New game: Dstroy

### ISSUE THREAD

**Issue by thp** [thp/dos-game-jam-demo-disc#21]
https://fully-bugged.itch.io/dstroy

via https://itch.io/post/11437700

**Comment by PyGuy-Programming** [comment]
@thp how can I add more games? Could u make a guide on this?

**Comment by thp** [comment]
> [@thp](https://github.com/thp) how can I add more games? Could u make a guide on this?

Add it to `catalog/gamectlg.csv` and rebuild.

**Comment by PyGuy-Programming** [comment]
yeah, i have problems with building it. like it doesnt work. i donr rember what all the issues where, but some where, that everything in layout r just broken symlinks.

**Comment by PyGuy-Programming** [comment]
e.g. catalog/layout/DEMO2023 is a symlink that expects the folder DEMO2023 to be in catalog/ which it isnt

**Comment by thp** [comment]
The `DEMO2023` folder contains all the games' files. It's not included in the Git repository, because it's huge.

It can be obtained from the ISO: https://thp.itch.io/djg-demo-disc-2023
